calculator.py

```diff
@@ -38,6 +37,12 @@ def tan(self, x):
 
         return math.tan(x)
 
+    def sin(self, x):

```

Comment:
Change this to:

`def sin(self, xInRadians):` 

and update the documentation and usage of `x`